docs/access_rights/organization_account.md

```diff
@@ -8,53 +8,54 @@ paths:
 
 # Organization account
 
-> **Warning** <br/>
-> The organization account is currently in the **closed beta testing** stage and available only to users with [**business**](https://apify.com/pricing) or higher subscriptions.<br/>
-> If you are interested in joining the closed beta testing, contact us at [support@apify.com](mailto:support@apify.com?subject=Organization%20account%20beta%20testing).
+> The organization account is only available to accounts using our new billing system. If you see the migration message when you [log in](https://my.apify.com), you will need to migrate.
 
-Apify's organization account allows multiple team members to collaborate on projects. It enables you to manage your team members' [permissions]({{@link access_rights/list_of_permissions.md}}) and to centralize your billing.
+Organization accounts allow groups to collaborate on projects. It enables you to manage your team members' [permissions]({{@link access_rights/list_of_permissions.md}}) and to centralize your billing without having to share the credentials of a single personal account.
 
-At the same time, a separate organization account means teams no longer have to share the credentials of a single personal account. You can [switch]({{@link access_rights/organization_account/how_to_use.md#in-the-apify-app}}) between your personal and organization accounts seamlessly in just two clicks.
+You can [switch]({{@link access_rights/organization_account/how_to_use.md#in-the-apify-app}}) between your personal and organization accounts in just two clicks: in the [Apify app](https://my.apify.com), click the account button in the top-right corner, then select the organization.
```

Comment:
"top-right corner"

actually, this will be changed with new menu in the next few days (probably this week)

![image](https://user-images.githubusercontent.com/697301/105210985-07b6e180-5b4c-11eb-9bd2-d55aacde3e9a.png)